.github/workflows/go_tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           fetch-depth: 2
       - uses: actions/setup-go@v3
         with:
-          go-version: '1.18'
+          go-version: '1.19'
```

Comment:
Should we still test on the oldest version of Go that we support?

Reply:
I replied and wasn't aware of Danny's response. Certainly can't do matrix testing along with the format check as evidenced by the failure with this 1.19 change, but if that's split out, then I don't have a problem with.

I am against doing so at the same time as a routine upgrade. We should file a p3 issue to do the matrix testing change if we're keen on it, and separately for fixing the formatting for 1.19.  1.19 introduces newer interpretations of the Go Doc comments which allows for richer linkified documentation. I'll file the formating fix issue.